packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -880,16 +880,32 @@ export const GenericDashboardChartTile: FC<
     error,
     ...rest
 }) => {
-    if (isLoading)
+    const { projectUuid } = useParams<{
+        projectUuid: string;
+        dashboardUuid: string;
+    }>();
+
+    if (isLoading) {
         return (
             <TileBase
                 isEditMode={isEditMode}
+                chartName={tile.properties.chartName ?? ''}
+                titleHref={`/projects/${projectUuid}/saved/${tile.properties.savedChartUuid}/`}
+                description={''}
+                belongsToDashboard={tile.properties.belongsToDashboard}
                 tile={tile}
-                isLoading={true}
-                title={''}
+                isLoading
+                title={tile.properties.title || tile.properties.chartName || ''}
+                extraMenuItems={
+                    tile.properties.savedChartUuid && (
+                        <EditChartMenuItem tile={tile} />
+                    )
+                }
```

Comment:
would any of these changes affect current `enterprise` expectations @ZeRego ?

Reply:
We should check user permissions to show this option and if is minimal view.

Reply:
Thanks! Having a look !